### PR TITLE
PyCharm setup instructions

### PR DESCRIPTION
**Pull request by raccube**

### CODE REVIEW

File: programming/editors/pycharm.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<div class="info">
+These instructions should also apply to IntelliJ IDEA with the Python plugin installed although the location of Python SDK settings will be different.
+</div>
+
+In order to take full advantage of PyCharm's smart assistance features, such as code completion, PyCharm needs to have access to the libraries used in your code.
+The following instructions will guide you through setting up your local environment to have access our `sr.robot3` API.
+
+1. Open your robot code project in PyCharm.
+2. Head to `File` &rarr; `Settings` or `PyCharm` &rarr; `Preferences` on a Mac.
```
Comment (PeterJCLaw) [inline programming/editors/pycharm.md:16]: I suggest that we not use `code` style highlighting for menu items. For the VSCode article, the pattern was to **embolden** them instead. I realise this is a small thing, but we should probably be consistent.

The arrow is probably nicer than the ">" used in the VSCode article though, perhaps we should update that to use `&rarr;` too?

File: programming/editors/pycharm.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+![Location of Settings menu option in PyCharm for Windows and Linux]({{ site.baseurl }}/images/content/programming/pycharm_app_menu.png)
+3. Select `Project` &rarr; `Project Interpreter` from the list on the left.
+4. Click the `+` symbol at the top of the packages list to install a new library.
+![PyCharm Interpreter Settings dialog]({{ site.baseurl }}/images/content/programming/pycharm_interpreter_settings.png)
+5. Search for `sr.robot3` then click `Install Package`
+![Installing sr.robot3 package from within PyCharm]({{ site.baseurl }}/images/content/programming/pycharm_pkgs.png)
+If this results in an error, try selecting the `Install to user's site packages directory` checkbox at the bottom of the dialog and try again.
+6. All done. You can now close the Settings window.
+
+You should now be able to use code completion and have quick access documentation by pressing <kbd>Ctrl</kbd><kbd>Q</kbd> (or <kbd>⌃</kbd><kbd>J</kbd> on a Mac)
```
Comment (PeterJCLaw) [inline programming/editors/pycharm.md:26]: Does <kbd>^</kbd> really mean that here? Or does it mean pressing the Shift key? If the latter I suggest using one of <kbd>⇑</kbd> or <kbd>⇪</kbd> (or even just <kbd>Shift</kbd>).

**Comment by PeterJCLaw** [comment]
Generally this looks good, though I don't have PyCharm to hand so I'm not in a position to check the steps work. It would be great to do that before we approve this though. Maybe post in Slack to see if someone who has PyCharm could help?

File: programming/editors/pycharm.md (hunk 2 above)
Comment (raccube) [reply programming/editors/pycharm.md:26]: `^` refers to Control. This follows the general pattern of how macOS displays keyboard shortcuts. (The symbols are on the keycaps of Apple keyboards)
image

File: programming/editors/pycharm.md (hunk 1 above)
Comment (raccube) [reply programming/editors/pycharm.md:16]: I wonder if the arrow is more screen reader friendly

Comment (raccube) [reply programming/editors/pycharm.md]: 41af49ad

File: programming/editors/pycharm.md (hunk 2 above)
Comment (PeterJCLaw) [reply programming/editors/pycharm.md:26]: Ah, ok :+1:

**Comment by antoinepetty** [comment]
Looks good apart from that

File: programming/editors/pycharm.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+These instructions should also apply to IntelliJ IDEA with the Python plugin installed although the location of Python SDK settings will be different.
+</div>
+
+In order to take full advantage of PyCharm's smart assistance features, such as code completion, PyCharm needs to have access to the libraries used in your code.
+The following instructions will guide you through setting up your local environment to have access our `sr.robot3` API.
+
+1. Open your robot code project in PyCharm.
+2. Head to **File** &rarr; **Settings** or **PyCharm** &rarr; **Preferences** on a Mac.
+![Location of Settings menu option in PyCharm for Windows and Linux]({{ site.baseurl }}/images/content/programming/pycharm_app_menu.png)
+3. Select `Project` &rarr; `Project Interpreter` from the list on the left.
```
[the remark this answers is not in the document]
Comment (raccube) [reply programming/editors/pycharm.md:18]: Ah yes, this has changed since writing this. I should probably retake the screenshot as well.

**Pull request merged**